runtime-config.md

```diff
@@ -36,6 +36,15 @@ Only [mounts from the portable config](config.md#mount-points) will be mounted.
     }
 }
 ```
+Note: The following filesystems MUST be made available in each Linux application's filesystem
```

Comment:
@wking I thinks describe this after the mounts Exmaple for linux will exactly tell people to make sure these filesystems related mounts should be configured in runtime.json